tpp-app/src/services/utils/helpers.js

```diff
@@ -100,10 +98,10 @@ export const getCalendarByYear = async (year) => {
  * @param {Number} month number (January = 1)
  * @param {Number} year number
  */
-export const getSymptomsFromCalendar = async (calendar, day, month, year) => {
+export const getSymptomsFromCalendar = (calendar, day, month, year) => {
```

Comment:
ah you are right this doesn't need to be async. I'd say don't make the swap in this PR, and open an issue for removing the async keyword & updating all uses. Since this function gets called in a lot of different places, and also in other PRs that are currently open (mine, Emily's at least). 

What do you think @faye-tan

Reply:
I'm ok with changing it here, and I'll go through all the other PRs to ask you and Emily to change it before you merge since its true it doesn't have to be async

Reply:
ahh I like that suggestion. It was just that my code was bugging when it was async for some reason. I'll try it out with it back as an async function.

Reply:
oh ok!